### PR TITLE
Android l10n + langpacks link

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

**Review by jcristau — APPROVED** [review]
awesome, thank you

File: how-tos/l10n/android-l10n.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Android L10n
+============
+
+Similar to desktop's :ref:`l10n_cross_channel`, `android-l10n-tooling <https://github.com/mozilla-l10n/android-l10n-tooling/>`__ takes en-US strings from mobile repos and lands them in quarantine branches in the `android-l10n repo <https://github.com/mozilla-l10n/android-l10n>`__. For example, `this PR <https://github.com/mozilla-l10n/android-l10n/pull/483>`__.
```
Comment (JohanLorenzo) [inline how-tos/l10n/android-l10n.rst:4]: Question: would you happen to know what these branches are label `quaratine`?

**Review by JohanLorenzo — APPROVED** [review]
Looks great to me, thanks! 👍

File: how-tos/l10n/android-l10n.rst (hunk 1 above)
Comment (escapewindow) [reply how-tos/l10n/android-l10n.rst:4]: Good question :)
Most likely for the same reason we push to a `quarantine`-named repo in [cross-channel](https://docs.mozilla-releng.net/en/latest/how-tos/l10n/l10n-cross-channel.html) -- the changes are there for a human to inspect and push to the real strings location. I'll add that to this PR before merging.

**Pull request merged**